components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/Events/PendingAlarmsOptionsModel.cs

```diff
@@ -49,6 +49,18 @@ public class PendingAlarmsOptionsModel {
                 TimeSpan.FromMilliseconds(SnapshotInterval.Value) : (TimeSpan?)null;
         }
 
+        /// <summary>
+        /// ConditionIdIndex
+        /// </summary>
+        [IgnoreDataMember]
+        public int ConditionIdIndex { get; set; } = -1;
```

Comment:
These could be nullable uints. `uint?`. Would be cleaner to check for null than -1.

Reply:
Had to make then int?, sinces values[xx] required int. Didn't want to have a cast there